### PR TITLE
refactor: responsive text css helper

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/g354mc

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by alexbarnsley** [comment]
@ItsANameToo updated

**Comment by ItsANameToo** [comment]
it's like you read my mind @alexbarnsley

File: resources/assets/css/_general.css [hunk 1]
```diff
@@ -229,3 +229,10 @@ body {
     top: 0;
     width: 100%;
 }
+
+/* Child text wrapping */
+@variants responsive {
+    .no-wrap-children > span {
```
Comment (ItsANameToo) [inline resources/assets/css/_general.css:235]: should we make it clear it's only `span` children? 🤔

Comment (alexbarnsley) [reply resources/assets/css/_general.css]: renamed to `no-wrap-span-children`

Comment (alexbarnsley) [reply resources/assets/css/_general.css]: @ItsANameToo

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**